pages/common/ocamlc.md

```diff
@@ -0,0 +1,8 @@
+# ocamlc
+
+> The OCaml bytecode compiler.
+> Produces executables runnable by OCaml interpreter.
+
+- Create named executable from source file:
+
+`ocamlc -o {{output_name}} {{source_file.ml}}`
```

Comment:
Same as your other PR. Perhaps we could tweak this to make it more consistent with other pages?
```suggestion
`ocamlc -o {{path/to/output}} {{path/to/source_file.ml}}`
```

 - Are there any other examples we could include?
 - Is there any official documentation we can link to?

Reply:
Also, we should include the default usage of `ocamlc` without any flags. And let's name path/to/output as path/to/binary.